command-expansion-server/src/app.ts

```diff
@@ -54,7 +54,7 @@ type Context = {
   expansionDataLoader: InferredDataloader<typeof expansionBatchLoader>;
 };
 
-app.use(async (req: Request, res: Response, next: NextFunction) => {
+app.use(async (_: Request, res: Response, next: NextFunction) => {
```

Comment:
More of a general question. How come `req` -> `_`

Is it because it isn't being used?

Reply:
Looks like it, renaming unused variables to `_` gets rid of a linter warning for me. Curious if we should adopt this practice elsewhere

Reply:
Yeah this is a widely used pattern in JS apps - especially when using callback based APIs where you do not control what parameters are in the callback.